Lib/idlelib/pyparse.py

```diff
@@ -101,46 +101,19 @@
 """, re.VERBOSE).match
 
 
-class StringTranslatePseudoMapping(Mapping):
-    r"""Utility class to be used with str.translate()
+class ParseMap(dict):
+    r"""Dict subclass that maps anything not in dict to 'x'.
```

Comment:
Thought of this while I was sleeping.  Should be `class _ParseMap`?

Reply:
No.  PEP434 declares nearly all of idlelib 'private', and it became such in practice nearly 3 years later, for  in 3.6.  There is no point to further privatization.

Reply:
Thanks.